packages/runtime/container-runtime/api-report/container-runtime.api.md

```diff
@@ -268,7 +268,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents
     generateDocumentUniqueId(): string | (number & {
         readonly SessionUnique: "cea55054-6b82-4cbf-ad19-1fa645ea3b3e";
     } & {
-        readonly OpNormalized: "9209432d-a959-4df7-b2ad-767ead4dbcae";
+        readonly OpNormalized: "9209432d-a959-4df7-b2ad-767ead4dbcae"; /**
```

Comment:
This doesn't seem relevant to this PR?  Also formatting seems weird, seems the comment should be above the property?

Reply:
API report file, api-extractor does some weird things with comments sometimes. I didn't track what was causing this one though.

Reply:
Build fails if I do not include it (I've tried)